azure-devops/projects/cgn-portale-esercenti-projects/service_connections.tf

```diff
@@ -26,6 +50,20 @@ resource "azuredevops_serviceendpoint_github" "io-azure-devops-github-rw" {
   }
 }
 
+# Github service connection (pull request)
+resource "azuredevops_serviceendpoint_github" "io-azure-devops-github-pr" {
+  depends_on = [azuredevops_project.project]
```

Comment:
Not blocking, but I don't think it's required to add this explicit dependency: it's already implicit within the next line.